embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/internal/spans/CurrentSessionSpanImpl.kt

```diff
@@ -32,7 +34,9 @@ internal class CurrentSessionSpanImpl(
     /**
      * The span that models the lifetime of the current session or background activity
      */
-    private val sessionSpan: AtomicReference<EmbraceSpan?> = AtomicReference(null)
+    private val sessionSpan: AtomicReference<PersistableEmbraceSpan?> = AtomicReference(null)
+
+    private val sessionIdAttributeName = "session_id".toEmbraceAttributeName()
```

Comment:
I'm doing something to make us not have to hardcode this at the source of usage. Merge it like this and I'll fix it up